web/pdf_outline_viewer.js

```diff
@@ -156,9 +156,9 @@ class PDFOutlineViewer {
   /**
    * @param {PDFOutlineViewerRenderParameters} params
    */
-  render(params = {}) {
-    var outline = params.outline || null;
-    var outlineCount = 0;
+  render({ outline, }) {
+    outline = outline || null;
```

Comment:
Please refer to https://github.com/mozilla/pdf.js/pull/8594#discussion_r125156197, since that applies here as well.